app/views/home/index.html.erb

```diff
@@ -68,7 +68,7 @@
             <% if course.nil?%>
             <% else %>
               <p><h4><%= link_to "#{course.course_name + "-" + course.section.to_s + "-" + course.semester}", course, :method => :get %></h4></p>
-              <p><b> # of Students:</b> <%= Student.where(course_id:course.id, teacher:@id).count %> </p>
+              <p><b> # of Students:</b> <%= StudentCourse.where(course_id:course.id).count %> </p>
```

Comment:
Should we add a unique check here? To make sure it doesn't count the same student in several classes as multiple students?

Reply:
good catch!

Reply:
I'll add it

Reply:
It's unique, course id should be unique for a student (even if the student is in multiple classes)